tests/php/Authenticator/LoginHandlerTest.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace SilverStripe\MFA\Tests\Authenticator;
+
+use SilverStripe\Control\HTTPResponse;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\FunctionalTest;
+use SilverStripe\MFA\Authenticator\MemberAuthenticator;
+use SilverStripe\MFA\Method\MethodInterface;
+use SilverStripe\MFA\Model\RegisteredMethod;
+use SilverStripe\MFA\Service\MethodRegistry;
+use SilverStripe\MFA\Store\SessionStore;
+use SilverStripe\MFA\Tests\Stub\BasicMath\Method;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+class LoginHandlerTest extends FunctionalTest
+{
+    protected static $fixture_file = 'LoginHandlerTest.yml';
+
+    protected function setUp()
+    {
+        parent::setUp();
+        Config::modify()->set(MethodRegistry::class, 'methods', [Method::class]);
+
+        Injector::inst()->load([
+            'SilverStripe\Security\Security' => [
```

Comment:
We could use `Session::class` here, but not a biggy

Reply:
`Security::class` - Thanks.